### ISSUE TITLE
[Term Entry] Swift Arrays: `.first`

### ISSUE THREAD

**Issue by yangc95** [Codecademy/docs#1106]
### Reason/inspiration (optional)

Please write a new entry under:
content/swiftui/concepts/views/terms/ about the Text view in SwiftUI.

A "Syntax" section with pseudo code snippet.
An "Example" section with a swift code snippet and an accompanying image to display the view.

To get started with the PR and entry file:

Under content/swift/concepts/arrays/terms/, create a new folder called first.
Then create a new file called first.md under content/swift/concepts/arrays/terms/first

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

**Comment by yangc95** [comment]
A note that the arrays.md file mentions the `.first` property. These are different from methods and this may require a discussion to see if it is okay to mix properties and methods under array terms.

**Comment by SSwiniarski** [comment]
@yangc95 I think we have to place both under terms, since we don't really have another option. I'm also not sure if I like mixing the property and method entries.

**Comment by Dusch4593** [comment]
I agree with @SSwiniarski, I think that there isn't a more suitable location/arrangement as of now. For a (hopefully near-)future version, I'd like to see a separate tables for methods and properties under certain concepts like Arrays, Strings, etc.

**Comment by cslylla** [comment]
Hi, I have seen that this issue hasn't been assigned to anyone yet. I would like to work on this issue. I can write the  `.first`  array property.
